plugin/trino-oracle/src/main/java/io/trino/plugin/oracle/OracleClient.java

```diff
@@ -204,7 +204,7 @@ protected String generateTemporaryTableName()
     protected void renameTable(ConnectorSession session, String catalogName, String schemaName, String tableName, SchemaTableName newTable)
     {
         if (!schemaName.equalsIgnoreCase(newTable.getSchemaName())) {
-            throw new TrinoException(NOT_SUPPORTED, "Table rename across schemas is not supported in Oracle");
+            throw new TrinoException(NOT_SUPPORTED, "This connector does not support renaming tables across schemas");
```

Comment:
as a followup please add a DROP schema test -- OracleClient lacks `dropSchema` override

Reply:
Oracle also lacks `createSchema`, I'm not sure how to test `dropSchema` without it.

Reply:
If a connector lacks `SUPPORTS_CREATE_SCHEMA`, can we assume it'll always lack drop schema support? I can add another negative assert in `BaseConnectorSmokeTest.testCreateSchema()`

Reply:
Generally no connector should support DROP SCHEMA but not CREATE (and actually vice versa). Users will expect the two to go in pair.

(If a connector insist about being _special_, it may end up needing to override a test; this always have been and remains an option)

Reply:
Added as a separate commit.